go/contacts/cache.go

```diff
@@ -12,13 +15,121 @@ type CachedContactsProvider struct {
 	Provider ContactsProvider
 }
 
-func (c *CachedContactsProvider) LookupPhoneNumbers(mctx libkb.MetaContext, numbers []keybase1.RawPhoneNumber,
-	userRegion keybase1.RegionCode) (res []ContactLookupResult, err error) {
-	return c.Provider.LookupPhoneNumbers(mctx, numbers, userRegion)
+var _ ContactsProvider = (*CachedContactsProvider)(nil)
+
+type CachedLookupResult struct {
+	ContactLookupResult
+	Resolved bool
+	cachedAt time.Time
+}
+
+type ContactsCache struct {
+	Lookups map[string]CachedLookupResult
+	Version struct {
```

Comment:
what is minor? backwards compatible changes?

Reply:
yes, i felt inspired by something that Miles did in team cache where he could update team cache to new version using info that was already there: https://github.com/keybase/client/pull/17353/files#diff-4c4b8b19dca791c68066641d12b1df46R146

I don't know if it ever becomes useful though.